machine_management/cluster_api_machine_management/cluster-api-about.adoc

```diff
@@ -9,7 +9,7 @@ toc::[]
 :FeatureName: Managing machines with the Cluster API
 include::snippets/technology-preview.adoc[]
 
```

Comment:
This may be a dumb question, but why is the TP warning first, before the description?